src/tape/ensemble.py

```diff
@@ -1142,6 +1160,65 @@ def s2n_inter_quartile_range(flux, err):
                 meta=meta,
             )
 
+        # Output standardization
+        if isinstance(batch, EnsembleSeries):
+            if batch.name == self._id_col:
+                batch = batch.rename("result")
+            batch = EnsembleFrame.from_dask_dataframe(batch.to_frame())
+            if len(on) > 1:
+                batch = batch.reset_index()
```

Comment:
Should we here and further do it in-place? I guess that it wouldn't change performance for Dask Dataframes (while it could help with pandas), but I'm not sure...

Reply:
I think the answer is no. For one it's not supported in Dask, but also it even really isn't advisable in Pandas it seems... https://stackoverflow.com/questions/43893457/understanding-inplace-true-in-pandas

Reply:
Thank you, sometimes Pandas' interfaces has so little sense...